### PR TITLE
Update `td_load` and `td_ddl` docs and add S3 v2 sample for input

### PR DESCRIPTION
**Pull request by akito19**
This PR intends to make it easier to use `td_ddl` and `td_load` operators.

Required options is based on Treasure Data production doc:
https://docs.treasuredata.com/display/public/INT/Treasure+Data+Integrations+by+Alphabetical+Sort

### CODE REVIEW

**Comment by akito19** [comment]
@treasure-data/support-engineer Could someone review it?

File: td_load/s3_v2/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+The workflow also uses [Secrets](https://docs.treasuredata.com/display/public/PD/Workflows+and+Machine+Learning-secrets) feature, so that you don't have to include your datasource credentials to your workflow files.
+
+## Content
+
+This directory has 2 types of workflows. These are different from authentication method.
+
+One uses a secret key, which is the authentication method until now.
+The other uses [AssumeRole (AWS documentation)](https://docs.aws.amazon.com/STS/latest/APIReference/API_AssumeRole.html).
+
+S3 v2 intoroduced a newly authentication method with AssumeRole. If you want use the method, you need to v2 integration instead of v1.
```
Comment (mickeey2525) [inline td_load/s3_v2/README.md:16]: > you need to v2 integration instead of v1.

use is missing?

`you need to use v2 integration instead of v1.`

**Review by mickeey2525 — APPROVED** [review]
Just a minor comment.

File: td_load/s3_v2/README.md (hunk 1 above)
Comment (akito19) [reply td_load/s3_v2/README.md]: Thanks. Fixed this.

**Pull request merged**